src/main/steps/applicant1/app-represented/content.ts

```diff
@@ -0,0 +1,138 @@
+import config from 'config';
+
+import { Applicant2Represented, State } from '../../../app/case/definition';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { SupportedLanguages } from '../../../modules/i18n';
+import { isCountryUk } from '../../applicant1Sequence';
+import type { CommonContent } from '../../common/common.content';
+import { formattedCaseId } from '../../common/content.utils';
+import { currentStateFn } from '../../state-sequence';
+import { getProgressBarContent } from '../hub-page/progressBarLabels';
+
+const en = (
+  { userCase, partner, isJointApplication, webChat, openingTimes, telephoneNumber }: CommonContent,
+  feedbackLink: string
+) => ({
+  title: 'Application represented',
+  appRepresentedText: `${
+    userCase.applicant1SolicitorRepresented
+      ? 'You no longer have access to this case due to informing the Court that you are represented. Please contact your Solicitor at your earliest convenience.'
+      : ''
```

Comment:
We might be able to remove this condition if they only reach this page when represented?

Reply:
erm, yep